apis/comms_api/scripts/wazuh_comms_apid.py

```diff
@@ -199,6 +196,13 @@ def load(self):
     signal.signal(signal.SIGINT, signal.SIG_IGN)
 
     try:
+        app = FastAPI()
+        app.add_middleware(SecureHeadersMiddleware)
+        # TODO: decide whether to use GZip or Brotli (latter requires https://github.com/fullonic/brotli-asgi)
+        app.add_middleware(GZipMiddleware)
```

Comment:
Gabriel mentioned he prefers Brotli because it's more efficient and has smaller results, but that would require introducing an external dependency or handle the compression ourselves. So I used gzip until we confirm

Reply:
Now is the moment to decide this. We must confirm with the management team and make the necessary changes if Brotly is chosen.

Reply:
As far as I'm concerned there's no definitive decision yet, but all teams are trying to use Brotli but it might get reverted in the future. I decided to use the `brotli-asgi` dependency which implements a minimal Brotli middleware and has gzip as a fallback mechanism. I've also updated the test example from the description accordingly.